pkg/apis/networking/sets.go

```diff
@@ -114,3 +114,115 @@ func (s GroupMemberPodSet) IsSuperset(o GroupMemberPodSet) bool {
 func (s GroupMemberPodSet) Equal(o GroupMemberPodSet) bool {
 	return len(s) == len(o) && s.IsSuperset(o)
 }
+
```

Comment:
not sure if the following is required as it is already declared under v1beta1/sets.go

Reply:
If you look at GroupMemberPodSet it is also defined in both networking/sets.go and v1beta1/sets.go. They might be able to be merged via imports but I am not sure of the scope